class_person.rb

```diff
@@ -1,12 +1,8 @@
-class Person < Nameable
-  attr_accessor :name
+class Person
+  attr_accessor :name, :rentals
 
   def initialize(name)
-    super()
     @name = name
-  end
-
-  def correct_name
-    @name
+    @rentals = []
   end
 end
```

Comment:
- [ ] To correctly match the [expected requirements](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_associations.md) for the `Person` class, kindly ensure that:
- You don't delete the previous code lines as you will need them in your next projects in order to complete your app.
- you should have a method for adding `rentals` where the person itself is sent as a parameter to create the new rental achieving the 'has-many' association`. Your class should look similar to this:
```ruby
class Person
  attr_accessor :name, :age 
  attr_reader :id 

  def initialize(age, name = "Unknown", parent_permission = true)
    @id = Random.rand(1..1000)
    @age = age
    @name = name
    @parent_permission = parent_permission
    @rentals = []
  end

  def can_use_services? 
    of_age?() || @parent_permission
  end

  def add_rental(book, date)
    Rental.new(date, book, self)
  end
 
  private 

  def of_age?
    @age >= 18
  end
end
```